src/components/Tooltip.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+
+function Tooltip({ text, title, children }: { text: string[], title?: string, children: React.ReactNode }) {
+    const [visible, setVisible] = useState(false);
+
+    return (
+        <span
+            onMouseEnter={() => setVisible(true)}
+            onMouseLeave={() => setVisible(false)}
+            style={{ position: 'relative', cursor: 'pointer', display: 'flex', alignItems: 'center' }}
```

Comment:
Why not use tailwind css classes?

Reply:
Old habits die hard haha. I will update that.